core/src/main/java/org/apache/gravitino/cache/EntityCacheKey.java

```diff
@@ -41,7 +59,7 @@ public class EntityCacheKey {
    */
   public static EntityCacheKey of(
       NameIdentifier ident, Entity.EntityType type, SupportsRelationOperations.Type relationType) {
-    return new EntityCacheKey(ident, type, relationType);
+    return new EntityCacheKey(ident, type, relationType, null);
```

Comment:
What's the purpose of setting a Boolean field to null?

Reply:
Hi @yuqi1129 ,

Sorry that I thought we just needed to cache for three states. 😊
Please let me know whether the plan in the first comment matches our design.

Best Regards,
Edward